csv2sql.drush.inc

```diff
@@ -45,46 +46,53 @@ function drush_csv2sql($csv_path) {
     return;
   }
 
-  if (!file_exists($csv_path)) {
+  // Check if the command has a file direct path.
+  $files = $csv_path ? array($csv_path) : array();
+  // Check if the command has a directory path to scan.
+  if ($dir = drush_get_option('dir')) {
+    $files = csv2sql_scan_for_csv_files($dir);
+  }
+
+  if (!file_exists($csv_path) && empty($files)) {
     drush_print('Given file does not exist.');
```

Comment:
in case neither file-path nor directory given - message should be different. Something like "Please, provide path to your CSV file or to the directory"